### ISSUE TITLE
messaging

### ISSUE THREAD

**Issue by moeller0** [lynxthecat/cake-autorate#102]
https://github.com/lynxthecat/cake-autorate/blob/1233ece57e824f67218cadad87fa8356ac0cfea3/cake-autorate_launcher.sh#L18

I wonder whether we should move the termination message to after the wait? (Changing from "terminating" to "terminated" or "ended"). For my immediate debugging I wanted to see when the process of termination was initiated but it might be confusing to still find messages related to the terminating instances for other users?

**Comment by lynxthecat** [comment]
Would it be overkill to include both terminating and terminated? As it happens have just watched Terminator 1, 2, 3 and 4 and 5 is tonight.

**Comment by moeller0** [comment]
Oh, I am all for verbose reporting, but we are talking about the system log here and there folks are traditionally very peculiar. We can defend a single start and stop message but I think more will get us unhappy users...

The other question is, whether these messages should not be generated in cake-autorate.sh instead, to also trigger when people call cake-autorate.sh directly without the launcher? (Doing one-by-one termination still feels like the right thing as it makes debugging easier and will not really matter in regards to run-time).

**Comment by lynxthecat** [comment]
I think this is much more your area of expertise - what would you recommend?

**Comment by moeller0** [comment]
I think we should put the messages into cake-autorate.sh and only trigger the ones in cake-autorate_launcher.sh if a local do_debug variable is set. These might be relevant for debugging the launcher but do not appear of generic interest, so we could also remove these from the launcher until there is reason to debug it again (give the launchers nice simplicity that might be "never").

**Comment by lynxthecat** [comment]
Ah, so you mean we should remove the logging messages inside the launcher?

**Comment by moeller0** [comment]
Yes, I think we now have start and stop messages inside the script so we can remove them from the launcher. Yes these are slightly different in scope (the launcher has the earlier start the main script the later end message), but for informative logging both seem OK, and since the launcher is not mandatory, I think putting the logging into the main script is the better option.

**Comment by lynxthecat** [comment]
Removed with this commit: https://github.com/lynxthecat/cake-autorate/commit/531b356c2d341615af3085752b70fc908af55ead.
